.github/workflows/ci.yml

```diff
@@ -42,6 +42,12 @@ jobs:
       - name: Test
         run: deno test --unstable --allow-all
 
+      - name: Generate Node tests
```

Comment:
Do this ahead of time and commit the results.
We shouldn't be adding module specific tests to the CI.

Reply:
I don't do this for a simple reason, I don't want reviewers to be checking node test files every time one is enabled.

This also ensures that the final result will be in fact a Node test and not some modified version of one (that's the whole reason why there is an ignore system in those cases)